common/tier4_autoware_utils/include/tier4_autoware_utils/geometry/geometry.hpp

```diff
@@ -731,6 +735,70 @@ geometry_msgs::msg::Pose calcInterpolatedPose(
 
   return output_pose;
 }
+
+inline geometry_msgs::msg::Vector3 createVector3(const double x, double y, double z)
+{
+  return geometry_msgs::build<geometry_msgs::msg::Vector3>().x(x).y(y).z(z);
+}
+
+inline geometry_msgs::msg::Twist getTwist(
+  const geometry_msgs::msg::Vector3 & velocity, geometry_msgs::msg::Vector3 & angular)
+{
+  geometry_msgs::msg::Twist twist;
+  twist.linear = velocity;
+  twist.angular = angular;
+  return twist;
+}
+
+inline geometry_msgs::msg::TwistWithCovariance getTwistWithCovariance(
+  const geometry_msgs::msg::Twist & twist)
+{
+  geometry_msgs::msg::TwistWithCovariance twist_with_covariance;
+  twist_with_covariance.twist = twist;
+  return twist_with_covariance;
+}
+
+inline geometry_msgs::msg::TwistWithCovariance getTwistWithCovariance(
+  const geometry_msgs::msg::Vector3 & velocity, geometry_msgs::msg::Vector3 & angular)
+{
+  geometry_msgs::msg::TwistWithCovariance twist_with_covariance;
+  twist_with_covariance.twist = getTwist(velocity, angular);
+  return twist_with_covariance;
+}
+
+inline double calcTwistNorm(const geometry_msgs::msg::Vector3 & v)
+{
+  return std::sqrt(v.x * v.x + v.y * v.y + v.z * v.z);
+}
+
+inline double calcTwistNorm(const geometry_msgs::msg::Twist & twist)
+{
+  return calcTwistNorm(twist.linear);
+}
+
+inline double calcTwistNorm(const geometry_msgs::msg::TwistWithCovariance & twist_with_covariance)
+{
+  return calcTwistNorm(twist_with_covariance.twist.linear);
```

Comment:
I don't think we need these wrappers in this library.
- `calcTwistNorm(twist.linear);`
- `calcTwistNorm(twist_with_covariance.twist.linear);`

Reply:
fixed at [eded17e](https://github.com/autowarefoundation/autoware.universe/pull/2049/commits/eded17e4d41c96b258d998abe0e96a8415daa453)